tests/integration/session_endianness/test.rflx

```diff
@@ -23,7 +23,7 @@ package Test is
 
       state Copy is
       begin
-         Out_Msg := Messages::Msg_LE'(B => In_Msg.A);
+         Out_Msg := Messages::Msg_LE'(C => In_Msg.A, D => In_Msg.B);
```

Comment:
This test only tests one direction. For example, it wouldn't be detected if parsing for LE would not work correctly.

Reply:
That's true. But wouldn't it be better to test that in unit tests if possible? The test here might get a bit unwieldy.

Reply:
done

Reply:
Isn't reading of LE messages still missing?

Reply:
Shouldn't the copy/copy2 states take care of that?

Reply:
as discussed via slack, it does not. Added another Read for an LE message.